tests/Doctrine/Tests/Common/Annotations/DocParserTest.php

```diff
@@ -1184,7 +1185,7 @@ class A {
 
         try {
             $parser = $this->createTestParser();
-            $parser->parse($docblock);
+            self::assertEmpty($parser->parse($docblock));
         } catch (\Exception $e) {
```

Comment:
catching any exception and then failing is not good: it swallows the clean failure exception.

would be much better to remove the try/catch entirely (same in previous test).